zuul.d/project-templates.yaml

```diff
@@ -1671,3 +1671,11 @@
     periodic:
       jobs:
         - build-ansible-collection
+
+- project-template:
+    name: ansible-collections-cloud-aws_troubleshooting
+    check:
+      jobs: &ansible-collections-cloud-aws_troubleshooting-jobs
+        - tox-ansible-lint
```

Comment:
I believe tt should be `ansible-tox-linters` actually.

Reply:
@goneri done, thanks!